### DISCUSSION TITLE
Mobile Web App Capable

### DISCUSSION THREAD

**Discussion by Arubinu** [skarab42/marv#174] [General]
En ajoutant ces quelques lignes au header du fichier `app/client/index.html`, on pourra profiter de la page web en fullscreen sur les appareils mobiles (tablettes/smartphones).

```
<meta name="apple-mobile-web-app-title" content="Marv" />
<meta name="apple-mobile-web-app-capable" content="yes" />
<meta name="apple-mobile-web-app-status-bar-style" content="black" />
<link rel="apple-touch-icon" href="icon.png" />
```

Lorsque l'on souhaite ajouter un raccourci d'un site web sur l'écran d'accueil d'un appareil mobile, celui-ci vérifie si c'est une WebApp via ces balises et autorise son utilisation en plein écran (sans la barre d'adresse).
J'ai remarqué que ce n'était pas fait disponible sur Marv, j'ai donc essayé et la manipulation a fonctionnée avec succès.

**Comment by batleforc** [comment]
Hello
De mémoire les tag que tu précise ne sont uttiliser que par les appareille Apple.
Je passe aussi par les [Manifest ](https://web.dev/add-manifest/)pour android.
`<meta name="mobile-web-app-capable" content="yes">` => serait un équivalent pour ta ligne 2.

**Reply by Arubinu** [reply]
Ok, mais je n'ai pas d'appareil Apple, et ca fonctionne :)
Skarab42 a dit qu'il voulait le faire via un manifest aussi pour que ce soit propre.
